code/modules/events/ion_storm.dm

```diff
@@ -54,11 +55,49 @@
 
 			log_game("Ion storm changed laws of [key_name(M)] to [english_list(M.laws.get_law_list(TRUE, TRUE))]")
 			M.post_lawchange()
+			
+		//Generate Cyborg law change
+	for(var/mob/living/silicon/robot/M in GLOB.alive_mob_list)
+		M.laws_sanity_check()
+		if(M.stat != DEAD && M.see_in_dark != 0)
```

Comment:
Why the see in dark?

Reply:
A few lines up the AI has the same check, so thats why that is there